### PR TITLE
Fix: Read addresses from db & config

### PR DESCRIPTION
**Pull request by rajeevbbqq**

### CODE REVIEW

File: airdrop/config.py [hunk 1]
```diff
@@ -24,6 +24,8 @@
 SIGNER_PRIVATE_KEY = 'AIRDROP_SIGNER_PRIVATE_KEY'
 SIGNER_PRIVATE_KEY_STORAGE_REGION = ''
 
+NUNET_TOKEN_ADDRESS = ''
```
Comment (anandrgitnirman) [inline airdrop/config.py:27]: Why is this blank if used

File: airdrop/infrastructure/repositories/airdrop_repository.py [hunk 2]
```diff
@@ -151,13 +151,22 @@ def register_airdrop_window_timeline(self, airdrop_window_id, title, description
...
+        return airdrop.contract_address
+
     def get_token_address(self, airdrop_id):
         airdrop = self.session.query(Airdrop).filter_by(id=airdrop_id).first()
 
         if airdrop is None:
             raise Exception("Airdrop not found")
 
-        return airdrop.contract_address
+        return airdrop.address
```
Comment (anandrgitnirman) [inline airdrop/infrastructure/repositories/airdrop_repository.py:169]: May be for future , address needs to be prefixed by what they mean

File: airdrop/config.py (hunk 1 above)
Comment (rajeevbbqq) [reply airdrop/config.py:27]: This file get replaced from S3 config when code build triggers

File: airdrop/infrastructure/repositories/airdrop_repository.py (hunk 2 above)
Comment (rajeevbbqq) [reply airdrop/infrastructure/repositories/airdrop_repository.py:169]: It will be addressed in a separate PR @anandrgitnirman

**Review by anandrgitnirman — APPROVED** [review]

**Pull request merged**